packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -16,6 +20,46 @@ export default class {
     return this.project;
   }
 
+  isManuallyReviewedByAntiFraud() {
+    return this.$http
+      .get(`/cloud/order/?planCode=project.2018`)
+      .then(({ data: orders }) => {
+        const validatingOrder = orders.find(
+          (order) =>
+            order.status.toLowerCase() ===
+            ORDER_FOLLOW_UP_STEP_ENUM.DELIVERING.toLowerCase(),
+        );
+
+        return validatingOrder
+          ? this.getOrderFollowUp(validatingOrder?.orderId)
+          : false;
+      })
+      .then((orderFollowUp) => {
+        if (orderFollowUp) {
+          const deliveringStep = orderFollowUp?.find(
+            (element) => element.step === ORDER_FOLLOW_UP_STEP_ENUM.VALIDATING,
+          );
+          return (
+            deliveringStep?.history.find(
+              (event) =>
+                event.label ===
+                ORDER_FOLLOW_UP_HISTORY_STATUS_ENUM.FRAUD_MANUAL_REVIEW,
+            ) || false
+          );
+        }
+        return false;
+      })
+      .catch(() => {
+        return false;
+      });
+  }
```

Comment:
The complexity of this method could be reduced
```suggestion
  isManuallyReviewedByAntiFraud() {
    return this.$http
      .get(`/cloud/order/?planCode=project.2018`)
      .then(({ data: orders }) => {
        const { DELIVERING, VALIDATING } = ORDER_FOLLOW_UP_STEP_ENUM;
        const { FRAUD_MANUAL_REVIEW } = ORDER_FOLLOW_UP_HISTORY_STATUS_ENUM;
        const { orderId } = orders.find(
          ({ status }) => status.toLowerCase() === DELIVERING.toLowerCase(),
        );
        return orderId
          ? this.getOrderFollowUp(orderId).then((orderFollowUp) =>
              orderFollowUp
                .find(({ step }) => step === VALIDATING)
                .history.some(({ label }) => label === FRAUD_MANUAL_REVIEW),
            )
          : false;
      })
      .catch(() => false);
  }
```